content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -185,7 +185,7 @@ another claim because the previous claimant's data remains on the volume.
 An administrator can manually reclaim the volume with the following steps.
 
 1. Delete the PersistentVolume. The associated storage asset in external infrastructure
-   (such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume) still exists after the PV is deleted.
+   (such as an AWS EBS or GCE PD volume) still exists after the PV is deleted.
```

Comment:
AWS EBS and GCE PD in-tree drivers are already removed.

Reply:
"AWS EBS"  in this line should be removed. I missed it.
GCE PD in-tree drivers are removed is k8s v1.28,  the main branch currently points to v1.27. It should be removed at the dev-1.28 branch, right?